crates/bevy_tasks/src/task_pool.rs

```diff
@@ -275,9 +275,8 @@ impl TaskPool {
         F: for<'scope> FnOnce(&'scope Scope<'scope, 'env, T>),
         T: Send + 'static,
     {
-        Self::THREAD_EXECUTOR.with(|scope_executor| {
-            self.scope_with_executor_inner(true, scope_executor, scope_executor, f)
-        })
+        let scope_executor = ThreadExecutor::new();
+        self.scope_with_executor_inner(true, &scope_executor, &scope_executor, f)
```

Comment:
The changes to this file look like a mistake? At the very least they shouldn't be in this PR.

Reply:
This tonemapping pr was brought to you by the `debug_asset_server`. I basically need this PR https://github.com/bevyengine/bevy/pull/7564 to be able to effectively iterate. I guess I could remove this and then not check the file in?